spec.md

```diff
@@ -2728,6 +2751,11 @@ extmodule = "extmodule" , id , ":" , [ info ] , newline , indent ,
               [ "defname" , "=" , id , newline ] ,
               { "parameter" , id , "=" , ( string | int ) , newline } ,
             dedent ;
+intmodule = "intmodule" , id , ":" , [ info ] , newline , indent ,
+              { port , newline } ,
+              [ "intrinsic" , "=" , id , newline ] ,
+              { "parameter" , "=" , ( string | int ) , newline } ,
+            dedent ;
```

Comment:
```suggestion
intmodule = "intmodule" , id , ":" , [ info ] , newline , indent ,
              { port , newline } ,
              "intrinsic" , "=" , id , newline ,
              { "parameter" , "=" , ( string | int ) , newline } ,
            dedent ;
```

As currently formulate, I think the `intrinsic` is mandatory as opposed to optional?

Reply:
yes